src/test/librbd/test_support.h

```diff
@@ -28,11 +28,11 @@ bool is_librados_test_stub(librados::Rados &rados);
 
 bool is_rbd_pwl_enabled(ceph::common::CephContext *ctx);
 
-#define REQUIRE(x) {			  \
-  if (!(x)) {				  \
-    std::cout << "SKIPPING" << std::endl; \
-    return SUCCEED(); 			  \
-  } 					  \
+#define REQUIRE(x) {							      \
+  if (!(x)) {								      \
+    std::cout << "SKIPPING TEST(REQUIRED FEATURE NOT ENABLED)" << std::endl; \
```

Comment:
I am not quite sure I like this. It looks too verbose as for a standard test message and it is used not only for features but also for config options.